client/src/App.css

```diff
@@ -40,3 +40,5 @@
 .read-the-docs {
   color: #888;
 }
+
+/* need to add css, background color, shading behind photos, form, buttons */
```

Comment:
best practice is to leave an empty line as the very last line of the file. Consider adding an empty line after this line

Reply:
Ah, yes, thank you for the reminder!